tests/python/relay/test_vm.py

```diff
@@ -595,7 +590,6 @@ def test_add_op_broadcast():
     test_let_scalar()
     test_compose()
     test_list_hd()
-    test_list_tl_empty_list()
```

Comment:
why remove the test?

Reply:
Because this one is supposed to fail.

Reply:
maybe it's a good idea to use `pytest.main()` in `__main__`?